README.md

```diff
@@ -57,7 +57,7 @@ async fn main() {
     // Put
     let key = b"test_key";
     let value = b"test_value";
-    kv_store.put(key, value).await;
+    kv_store.put(key, value).await.unwrap();
```

Comment:
how about use a `?` to replace `.unwrap()` in this sample code?

this requires a change from `async fn main()` into `async fn main() -> Result<()>`.